src/components/tabs/Tabs.js

```diff
@@ -0,0 +1,33 @@
+import { TabNav } from './TabNav';
+import TodoList from '../todoList';
+
+import './tabs.scss';
+
+export class Tabs extends Component {
+  state = {
+    selectedIndex: 0
+  }
+
+  selectTab = (event, selectedIndex) => {
+    event.preventDefault();
+    this.setState({ selectedIndex });
+  }
+
+  render() {
+    const { tabs, delTask, putTask } = this.props;
+    const { selectedIndex } = this.state;
+
+    return (
+      <div className="tabs">
+        <TabNav
+          tabs={tabs}
+          selectedIndex={selectedIndex}
+          selectTab={this.selectTab}
+        />
+        <div className="tab-content">
+          <TodoList task={tabs[selectedIndex]} delTask={delTask} putTask={putTask} />
+        </div>
```

Comment:
as I remember, we defined tabs content as:
```
<section className="tab-content">
  {tabs[selectedIndex].props.children}
</section>
```

`Tabs` is only wrapper for some specific content. But this content should be passed to the tabs through the props, but not inside Tabs component.
So, if I want to use your Tabs like
```
<Tabs>
  <Tab title="tab 1">
    Hello!
  </Tab>
<Tab title="tab 2">
    By!
  </Tab>
</Tabs>
```
it doesn't work with your changes